Tests/SonarScanner.MSBuild.PreProcessor.Test/RoslynAnalyzerProviderTests.cs

```diff
@@ -233,7 +233,7 @@ public void RoslynConfig_ValidProfile()
 
     #region Private methods
 
-    private List<SonarRule> createRules()
+    private List<SonarRule> CreateRules()
```

Comment:
Make all verbatim (xml) string in this file a raw string literals. There should be a built-in quickfix for those